src/player.cpp

```diff
@@ -12963,19 +12963,6 @@ void player::burn_move_stamina( int moves )
     }
 }
 
-field_id player::playerBloodType() const
-{
-    if (has_trait("ACIDBLOOD"))
```

Comment:
It duplicates `Character::bloodType()` verbatim. If we ever need a different behavior here, we can always override it.